### DISCUSSION TITLE
Unable to connect ms sql server when using DBeaver but SSMS works

### DISCUSSION THREAD

**Discussion by hieunm22** [dbeaver/dbeaver#23184] [DBeaver Development]
As title, I'm trying to connect microsoft sql server using Dbeaver 2019, with connection information as below, but it throws an error like this

![image](https://github.com/dbeaver/dbeaver/assets/5255023/efea7d7a-e90e-4e47-9301-5138ce809c0b)

![image](https://github.com/dbeaver/dbeaver/assets/5255023/01d34a7a-03dd-49bb-b3a7-5134cd7671a7)

but when switching to use SSMS to connect this database, it works

![image](https://github.com/dbeaver/dbeaver/assets/5255023/ac9511fa-8185-4022-a476-f1ac19248058)

could anyone tell me what am I missing is this a bug of dbeaver
thanks

**Comment by LonwoLonwo** [comment]
Hello @hieunm22 

You can take a look at these docs to fix your issue:
https://kb.sos-berlin.com/pages/viewpage.action?pageId=17499564
https://stackoverflow.com/questions/18841744/jdbc-connection-failed-error-tcp-ip-connection-to-host-failed

**Comment by hieunm22** [comment]
it works. thanks man. I miss to configure tcp/ip in sql configuration manager
